common/tools/dev-tool/src/templates/sampleReadme.md.ts

```diff
@@ -155,13 +155,13 @@ function createReadmeLink(info: SampleReadmeConfiguration) {
 /**
  * Creates a README for a sample package from a SampleReadmeConfiguration.
  */
-export default (info: SampleReadmeConfiguration): string => {
+export default async (info: SampleReadmeConfiguration): Promise<string> => {
   let stepCount = 1;
   const step = (content: string) => `${stepCount++}. ${content}`;
 
   const language = info.useTypeScript ? "TypeScript" : "JavaScript";
 
-  return format(
+  return await format(
```

Comment:
shouldn't need to await a promise before returning, can just return the format function and then you don't have to worry about making this function async either.